templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete a manufacturer</h1>
+  <p>Are you sure you want to delete a manufacturer: {{ manufacturer }}?</p> 
+  <p><i>All cars from this manufacturer will be also deleted!</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger" />
+    <button class="btn btn-success">
+              <a style="color: white" 
+                 href="{% url "taxi:manufacturer-list" %}">Cancel</a>
+    </button>

```

Comment:
The 'Cancel' button should not contain an anchor tag inside it. Instead, the 'button' element should have a 'type' attribute set to 'button' and the 'onclick' attribute should be used to redirect to the desired URL. This ensures proper button behavior and accessibility.